src/Dapr.Actors/Runtime/ReminderInfo.cs

```diff
@@ -55,28 +55,29 @@ internal static async Task<ReminderInfo> DeserializeAsync(Stream stream)
             var data = default(byte[]);
             int? repetition = null;
             TimeSpan? ttl = null;
-
-            if (json.TryGetProperty("dueTime", out var dueTimeProperty))
-            {
-                var dueTimeString = dueTimeProperty.GetString();
-                dueTime = ConverterUtils.ConvertTimeSpanFromDaprFormat(dueTimeString);
-            }
-
-            if (json.TryGetProperty("period", out var periodProperty))
-            {
-                var periodString = periodProperty.GetString();
-                (period, repetition) = ConverterUtils.ConvertTimeSpanValueFromISO8601Format(periodString);
-            }
-
-            if (json.TryGetProperty("data", out var dataProperty) && dataProperty.ValueKind != JsonValueKind.Null)
-            {
-                data = dataProperty.GetBytesFromBase64();
-            }
-
-            if (json.TryGetProperty("ttl", out var ttlProperty))
-            {
-                var ttlString = ttlProperty.GetString();
-                ttl = ConverterUtils.ConvertTimeSpanFromDaprFormat(ttlString);
+            if(json.ValueKind != JsonValueKind.Null){
```

Comment:
`DeserializeAsync<T>()` returns `object?`; would it make sense to also test `json` for `null`?

Reply:
Here we have been using `DeserializeAsync<JsonElement>` which returns `JsonElement` type, which is not a reference type and so can't be compared to null. It says this error on trying: `Operator '!=' cannot be applied to operands of type 'JsonElement' and '<null>'`.

Reply:
Handled null scenario differently.